libraries/state_history/log.cpp

```diff
@@ -386,7 +386,7 @@ chain::bytes state_history_chain_state_log::get_log_entry(block_num_type block_n
    }
 
    if (block_num < begin_block() || block_num >= end_block())
-      return {};
+      return {'\0'};
```

Comment:
also add the comment  `same as return fc::pack(std::vector<state_history::table_delta>{})`